src/test/ui/resolve/issue-14254.stderr

```diff
@@ -2,7 +2,10 @@ error[E0425]: cannot find function `baz` in this scope
   --> $DIR/issue-14254.rs:29:9
    |
 29 |         baz();
-   |         ^^^ did you mean `self.baz(...)`?
+   |         ^^^
+   |
+help: did you intend to call a method of the same name?
+   |         self.baz();
```

Comment:
Suggestions from resolve were intentionally reported "in place" as labels.
See https://github.com/rust-lang/rust/pull/38154, especially @jonathandturner's comments.
This is a pretty big regression IMO.